packages/interfaces/src/sharding.ts

```diff
@@ -1,12 +1,33 @@
-export type ShardInfo = {
-  clusterId: number;
-  shards: number[];
+/**
+ * Configuration for a Waku network. All nodes in a given network/cluster
+ * should have the same configuration.
+ */
+export type NetworkConfig = StaticSharding | AutoSharding;
+
+export type RelayShards = {
+  clusterId: ClusterId;
+  shards: ShardId[];
 };
 
 export type ContentTopicInfo = {
```

Comment:
I was thinking of removing it, it doesn't seem useful and if we want to use that it can be limited only to `IWaku` 
if you agree we can change todo to `remove` and later follow up

Reply:
I think I may be able to remove, let's see once tests pass